apps/storybook-react/tailwind.config.js

```diff
@@ -9,4 +9,5 @@ module.exports = {
     './src/**/*.{js,jsx,ts,tsx}',
     './stories/**/*.{js,jsx,ts,tsx}',
   ],
+  mode: 'jit',
```

Comment:
I believe JIT mode is the [default in v3](https://tailwindcss.com/blog/tailwindcss-v3#just-in-time-all-the-time) and above so this config option is deprecated and doesn't do anything. Have you read something different?